include/roboteam_ai/stp/Tactic.h

```diff
@@ -82,19 +67,19 @@ class Tactic {
     /**
      * Calls onInitialize of the tactic
      */
-    virtual void initialize() noexcept;
+    void initialize() noexcept;
 
     /**
      * Check if state machine is done, calls calculateInfoForSkill, calls update on the state machine with SkillInfo and calls onUpdate of this tactic for extra customization
      * @param info info passed by the Role
      * @return Status of the skill that is currently being ticked
      */
-    virtual Status update(StpInfo const &info) noexcept;
+    Status update(StpInfo const &info) noexcept;
 
     /**
      * Calls onTerminate
      */
-    virtual void terminate() noexcept;
+    void terminate() noexcept;
 
     /**
      * Ensure proper destruction of Tactic classes
```

Comment:
I'm not sure, but doesn't C++ need the virtual keyword to access the right method in the derived class, and will go to the base class method without it?

Reply:
`initialize(), terminate() and update()` are only called through the respective functions from Role, which are called from Play. These do not have overrides (and should not  have them either), so no need for virtual